cnames_active.js

```diff
@@ -629,6 +629,7 @@ var cnames_active = {
   "developerfolio": "saadpasta.github.io/developerFolio",
   "devilapi": "devilapi.kazult.repl.co",
   "devilowl": "website.kazult.repl.co",
+  "dev-panel": "https://github.com/Involve-Digital/dev-panel",
```

Comment:
```suggestion
  "dev-panel": "Involve-Digital.github.io/dev-panel",
```
If you're using GitHub Pages, this should be a github.io target (without https://)